scavenger-api/src/main/kotlin/com/navercorp/scavenger/repository/ExportSnapshotMethodRepository.kt

```diff
@@ -0,0 +1,37 @@
+package com.navercorp.scavenger.repository
+
+import com.navercorp.scavenger.entity.ExportSnapshotMethodEntity
+import com.navercorp.scavenger.spring.DelegatableJdbcRepository
+import org.springframework.data.jdbc.repository.query.Query
+import org.springframework.data.repository.query.Param
+import org.springframework.stereotype.Repository
+
+@Repository
+interface ExportSnapshotMethodRepository : DelegatableJdbcRepository<ExportSnapshotMethodEntity, String> {
+    @Query(
+        """
+             SELECT
+                s.filterInvokedAtMillis,
+                s.packages,
+                s.status,
+                s.excludeAbstract,
+                sn.parent,
+                sn.signature,
+                sn. `type`,
+                sn.usedCount,
+                sn.unUsedCount,
+                sn.lastInvokedAtMillis
+            FROM
+                snapshots AS s
+                JOIN snapshot_nodes AS sn
+            WHERE
+                s.id = :snapshotId
+                AND s.customerId = :customerId
+                AND s.id = sn.snapshotId
```

Comment:
```sql
             SELECT
                snapshots.filterInvokedAtMillis,
                snapshots.packages,
                snapshots.status,
                snapshots.excludeAbstract,
                snapshot_nodes.parent,
                snapshot_nodes.signature,
                snapshot_nodes. `type`,
                snapshot_nodes.usedCount,
                snapshot_nodes.unUsedCount,
                snapshot_nodes.lastInvokedAtMillis
            FROM
                snapshots
                INNER JOIN snapshot_nodes ON snapshots.id = snapshot_nodes.snapshotId
            WHERE
                snapshots.id = :snapshotId
                AND snapshots.customerId = :customerId
```
Please use `INNER JOIN` and the full table name to keep conventions.

Reply:
I fixed this review on commit 735e722.